src/main/java/mate/academy/model/Movie.java

```diff
@@ -5,8 +5,10 @@
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
+import mate.academy.lib.Inject;
 
 @Entity
+@Inject
```

Comment:
The `@Inject` annotation is not applicable to this class. It's used to mark dependencies that should be injected into a class, usually by a Dependency Injection framework like Spring or CDI. Please remove it.